docs/admin/kubeadm.md

```diff
@@ -4,1010 +4,22 @@ approvers:
 - luxas
 - errordeveloper
 - jbeda
-title: Kubeadm
+title: Overview of Kubeadm
 notitle: true
 ---
-# kubeadm Setup Tool Reference Guide
+# Overview of Kubeadm
 
-This document provides information on how to use kubeadm's advanced options.
+Kubeadm is a tool built to provide `kubeadm init` and `kubeadm join` as best-practice “fast paths” for creating kubernetes clusters. 
 
-* TOC
-{:toc}
+kubeadm performs the actions necessary to get a minimum viable cluster up and running. It only cares about bootstrapping, not about provisioning machines, by design. Likewise, installing various nice-to-have addons by default like the Kubernetes Dashboard, some monitoring solution, cloud provider-specific addons, etc. is not in scope. 
 
```

Comment:
Suggested change:
By design, it cares only about bootstrapping, not about provisioning machines. Likewise, installing various nice-to-have addons, like the Kubernetes Dashboard, monitoring solutions, and cloud-specific addons, is not in scope.